### PR TITLE
Use babel env usage & corejs@3

### LINKED ISSUE #691: Babel modules not found
With the update to 1.1.0 building the app fails with 

```
ERROR` in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/asyncToGenerator' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 108:42-92
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js

ERROR in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/typeof' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 108:8119-8159
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js

ERROR in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/regenerator' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 87:41-78
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! inventory@0.0.1 build: `npm run sprites && webpack --progress --hide-modules --config webpack.prod.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the inventory@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

v1.0.0 built fine. Do you have any idea what might have changed?

### PR DESCRIPTION
**Pull request by skjnldsv**
closes #692
Fixes #691

Tested on guests with this branch
Capture d’écran_2019-11-27_12-56-23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
 		"@babel/polyfill": "^7.4.4",
```
Comment (georgehrke) [inline package.json]: Can we remove "@babel/polyfill" now, since we switch to core-js?

Comment (skjnldsv) [reply package.json]: Yes! :)

Comment (skjnldsv) [reply package.json]: Done

**Comment by raimund-schluessler** [comment]
Works great, thanks. https://github.com/nextcloud/tasks/pull/688 is in now. 👍

**Pull request merged**